pages/common/cat.md

```diff
@@ -10,6 +10,14 @@
 
 `cat {{file1}} {{file2}} > {{target-file}}`
 
-- Append serveral files into the target file:
+- Append several files into the target file:
 
 `cat {{file1}} {{file2}} >> {{target-file}}`
+
+- Number the non-blank lines:
```

Comment:
Sounds like specialized behavior of the one below, wouldn't switching make more sense? Also I think you can drop "the".